Snail/Observable.swift

```diff
@@ -83,11 +83,11 @@ public class Observable<T> : ObservableType {
         scheduler.start()
 
         var next: T?
-        scheduler.event.subscribe(onNext: {
-            guard let event = next else {
+        scheduler.observable.subscribe(onNext: {
+            guard let nextValue = next else {
```

Comment:
I might prefer just `next` here. I like to name the unwrapped thing the same as the thing you're unwrapping

Reply:
See below comment